app/src/main/java/org/sinisterstuf/guesstheanimal/Greeting.java

```diff
@@ -1,34 +1,24 @@
 package org.sinisterstuf.guesstheanimal;
 
-import android.os.Bundle;
 import android.app.Activity;
 import android.content.Intent;
-import android.view.Menu;
+import android.os.Bundle;
```

Comment:
I was confused by this addition at first and now I see you've just re-ordered the imports.